src/bsp/src/posix/include/TCPUartMock.h

```diff
@@ -24,6 +25,8 @@ class TCPUartMock : public IHostUart {
   private:
     ILogger& m_logger;
 
+    BaseTask<configMINIMAL_STACK_SIZE * 100> m_listenTask;
```

Comment:
Was the x100 a test or do we really need such a big stack?

Reply:
It was the value that was there when I refactored it. I'll change it to something smaller

Reply:
my bad, it was probably a test from me